### PR TITLE
frontend for transferring to crypto wallet

### PR DESCRIPTION
**Pull request by jw122**
Allow transferring a merchant's account balance to any crypto wallet. Frontend counterpart to #19 
image

### CODE REVIEW

File: frontend/src/components/PaymentForm.tsx [hunk 1]
```diff
@@ -68,21 +68,17 @@ class PaymentForm extends Component {
...
-        }),
-        encryptionKeys: publicKey,
-      })
-      .then((ciphertext) => {
-        return {
-          encryptedCredentials: btoa(ciphertext),
-          keyId: pciEncryptionKey.keyId,
-        };
-      });
+    const options = {
```
Comment (jw122) [inline frontend/src/components/PaymentForm.tsx:71]: FYI -- this is some cleanup work I had to do with openpgp. I was likely using the wrong version/package before

File: frontend/src/components/TransferForm.tsx [hunk 2]
```diff
@@ -11,41 +11,37 @@ function TransferForm() {
...
+        amount: amountToSend,
+        currency: "USD",
+        destinationType: "blockchain",
+        destinationAddress: destinationAddress,
+        chain: "ETH",
+      })
+      .then((response) => {
+        console.log(
+          "Received successful response after transfer",
+          response.data
```
Comment (jw122) [inline frontend/src/components/TransferForm.tsx:29]: this always gets logged to the console, but I'm unable to get `response.data.transactionHash` for example, even when it's available
image

**Comment by popoaq** [comment]
Lgtm after merge conflict is resolved. Feel free to merge!

**Comment by jw122** [comment]
@popoaq thanks! fixed conflicts & merging

**Pull request merged**